.github/workflows/kind.yaml

```diff
@@ -85,14 +91,15 @@ jobs:
       - name: Set up external targets
         id: external_targets
         run: |
-          export worker2_ip=$(kubectl get nodes chart-testing-worker2 -o=jsonpath='{.status.addresses[?(@.type=="InternalIP")].address}')
-          export worker3_ip=$(kubectl get nodes chart-testing-worker3 -o=jsonpath='{.status.addresses[?(@.type=="InternalIP")].address}')
-          echo "worker2_ip=${worker2_ip}" >> $GITHUB_OUTPUT
-          echo "worker3_ip=${worker3_ip}" >> $GITHUB_OUTPUT
+          export worker6_ip=$(kubectl get nodes chart-testing-worker6 -o=jsonpath='{.status.addresses[?(@.type=="InternalIP")].address}')
+          export worker7_ip=$(kubectl get nodes chart-testing-worker7 -o=jsonpath='{.status.addresses[?(@.type=="InternalIP")].address}')
+          echo "worker6_ip=${worker6_ip}" >> $GITHUB_OUTPUT
+          echo "worker7_ip=${worker7_ip}" >> $GITHUB_OUTPUT
           kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.12.0/cert-manager.yaml
           kubectl rollout status -n cert-manager deployment.apps/cert-manager
           kubectl rollout status -n cert-manager deployment.apps/cert-manager-webhook
           kubectl create ns external-targets
+          cat .github/external-targets/certs.yaml | envsubst | kubectl apply -n external-targets -f - || true
```

Comment:
Why is this necessary?

Reply:
It was an ugly attempt to fix this [error](https://github.com/cilium/cilium-cli/actions/runs/8974239249/job/24648943491?pr=2496#step:11:78).
I'll address it in a separate PR, removed.